federation-2/router-bridge/src/planner.rs

```diff
@@ -693,6 +699,48 @@ pub struct IncrementalDeliverySupport {
     pub enable_defer: Option<bool>,
 }
 
+#[derive(Serialize, Debug, Clone, Default,PartialEq, Eq, Hash)]
+#[serde(rename_all = "camelCase")]
+/// Query planner debug configuration
+pub struct QueryPlannerDebugConfig {
+
+    /// If used and the supergraph is built from a single subgraph, then user queries do not go through the
+    /// normal query planning and instead a fetch to the one subgraph is built directly from the input query.
+    bypass_planner_for_single_subgraph: Option<bool>,
+
+
+    /// Query planning is an exploratory process. Depending on the specificities and feature used by
+    /// subgraphs, there could exist may different theoretical valid (if not always efficient) plans
+    /// for a given query, and at a high level, the query planner generates those possible choices,
+    /// evaluate them, and return the best one. In some complex cases however, the number of
+    /// theoretically possible plans can be very large, and to keep query planning time acceptable,
+    /// the query planner cap the maximum number of plans it evaluates. This config allows to configure
+    /// that cap. Note if planning a query hits that cap, then the planner will still always return a
+    /// "correct" plan, but it may not return _the_ optimal one, so this config can be considered a
+    /// trade-off between the worst-time for query planning computation processing, and the risk of
+    /// having non-optimal query plans (impacting query runtimes).
+    ///
+    /// This value currently defaults to 10 000, but this default is considered an implementation
+    /// detail and is subject to change. We do not recommend setting this value unless it is to
+    /// debug a specific issue (with unexpectedly slow query planning for instance). Remember that
+    /// setting this value too low can negatively affect query runtime (due to the use of sub-optimal
+    /// query plans).
+    max_evaluatedplans: Option<u32>,
```

Comment:
to camelcase to `maxEvaluatedPlans`, should this be:
```suggestion
    max_evaluated_plans: Option<u32>,
```